.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,30 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4.0.0
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@v0.1
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Update image tags
+        run: |
+          go install github.com/mikefarah/yq/v4@latest
```

Comment:
Since we are doing a python script anyhow, why not resolve this in python? so we don't need to do the yq part.

Reply:
See comment about maybe rewriting this whole feature to be all python later. It started life in bash/yq and then gained python just to do the editing without messing up the comments.